Bugsnag/Helpers/BSGRunContext.m

```diff
@@ -66,6 +70,14 @@ static void InitRunContext() {
         uuid_copy(bsg_runContext->machoUUID, image->uuid);
     }
     
+    if ([NSThread isMainThread]) {
+        bsg_runContext->isActive = GetIsActive();
+    } else {
+        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
```suggestion
        dispatch_sync(dispatch_get_main_queue(), ^{
```

should this block to wait for the context to be updated, or was the intent to update the context async (in which case more should move into the block (like updating the timestamp?)

Reply:
This is intentionally async, didn't want to block the calling thread.

The check is there because `GetIsActive()` must be called on the main thread.

Reply:
True, the timestamp could be a little behind, but I don't think it matters 🙃